hack/README

```diff
@@ -0,0 +1,13 @@
+This README documents how to generate CRD yaml files using ./hack/update-crd.sh
```

Comment:
when do you run "update-generated-code.sh"?

Can we potentially add a make rule that does both?

Reply:
absolutely, I will actually include a make rule in next PR to separate modules.
update-generated-code.sh should follow the similar pattern, it needs to run after any changes to types.go.

Reply:
Since both scripts are under the same folder, maybe put in instructions for both?  Since it's very likely that if you change types.go, you'll want to regenerate both

Reply:
SG. I will add it.
BTW, is it a standard to have a .md file for README? or its fine to leave it plain text.

Reply:
I think md would be useful